clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -1161,6 +1172,14 @@ void Clang::AddPreprocessingOptions(Compilation &C, const JobAction &JA,
   Args.AddLastArg(CmdArgs, options::OPT_C);
   Args.AddLastArg(CmdArgs, options::OPT_CC);
 
+  // When generating the using the integration footer, add the comments
```

Comment:
```suggestion
  // When generating using the integration footer, add the comments
```
or
```suggestion
  // When preprocessing using the integration footer, add the comments
```
or something of a sort :)